addons/project/models/res_company.py

```diff
@@ -15,11 +15,15 @@ class ResCompany(models.Model):
         help="Default Plan for a new analytic account for projects")
 
     def _compute_analytic_plan_id(self):
+        analytic_plan_projects = self.env.ref("analytic.analytic_plan_projects")
         for company in self:
             default_plan = self.env['ir.config_parameter'].with_company(company).sudo().get_param("default_analytic_plan_id_%s" % company.id)
             company.analytic_plan_id = int(default_plan) if default_plan else False
             if not company.analytic_plan_id:
-                company.analytic_plan_id = self.env['account.analytic.plan'].with_company(company)._get_default()
+                if analytic_plan_projects.sudo().company_id == company:
+                    company.analytic_plan_id = analytic_plan_projects.id
+                else:
+                    company.analytic_plan_id = self.env['account.analytic.plan'].with_company(company)._get_default()
```

Comment:
TBH, I did not know why we move `analytic.analytic_plan_projects` in real data. 🤔